client/post-editor/editor-ground-control/index.jsx

```diff
@@ -91,10 +93,23 @@ export default React.createClass( {
 			showDateTooltip: false,
 			firstDayOfTheMonth: this.getFirstDayOfTheMonth(),
 			lastDayOfTheMonth: this.getLastDayOfTheMonth(),
-			needsVerification: config( 'email_verification_gate' ) && userUtils.needsVerificationForSite( this.props.site ),
+			needsVerification: config( 'email_verification_gate' ) && this.props.userUtils.needsVerificationForSite( this.props.site ),
 		};
 	},
 
+	componentWillReceiveProps: function( nextProps ) {
+		this.setState( {
```

Comment:
I think setState already does the check, and only updates if it already has changed. Let me confirm that.